src/main/java/ch/ivyteam/ivy/maven/StopTestEngineMojo.java

```diff
@@ -83,7 +83,7 @@ public void execute() throws MojoExecutionException, MojoFailureException
   public EngineControl createEngineController() throws MojoExecutionException
   {
     File engineDir = getEngineDir(project);
-    if (!engineDir.exists())
+    if (engineDir == null || !engineDir.exists())
```

Comment:
Java does resolve if statements from left to right & stops as soon one evaluates to true, right? or am I confusing it with another language?

Reply:
yes looks good: however, if you can't test this scenario this code will very likely brake again in the future...